core/src/main/scala/precog/SbtPrecogBase.scala

```diff
@@ -233,7 +233,7 @@ abstract class SbtPrecogBase extends AutoPlugin {
           name = Some("Common sbt setup"),
           cond = Some("env.ENCRYPTION_PASSWORD != null")),
 
-      githubWorkflowBuild := WorkflowStep.Sbt(List("ci")),
+      githubWorkflowBuild += WorkflowStep.Sbt(List("ci")),
```

Comment:
This is incorrect. My PR does this correctly: it should be `:= Seq(WorkflowStep.Sbt(List("ci")))`, otherwise the build will run both `test` *and* `ci`.